predictionnet/utils/miner_hf.py

```diff
@@ -0,0 +1,94 @@
+import os
+
+import bittensor as bt
+from dotenv import load_dotenv
+from huggingface_hub import HfApi
+
+
+class Miner_HF_interface:
+    def __init__(self, config: "bt.Config"):
+        load_dotenv()
+        self.api = HfApi(token=os.getenv("MINER_HF_ACCESS_TOKEN"))
+        self.config = config
+
+        print(f"Initializing with config: model={config.model}, repo_id={config.hf_repo_id}")
+
+        if not hasattr(self.config, "model") or not hasattr(self.config, "hf_repo_id"):
+            raise ValueError("Config must include 'model' and 'hf_repo_id' parameters")
+
+    def upload_model(self, repo_id=None, model_path=None, hotkey=None):
+        repo_id = repo_id or self.config.hf_repo_id
+        model_path = model_path or self.config.model
+
+        print(f"Trying to upload model: repo_id={repo_id}, model_path={model_path}, hotkey={hotkey}")
```

Comment:
Similar to the above. I don't think we want actual print statements here. I won't comment on the other print statements below